challenge-11/challenge-11.js

```diff
@@ -1,54 +1,91 @@
-// Envolva todo o código desse arquivo em uma IIFE (incluindo esse comentário).
+(function(){
+	// Envolva todo o código desse arquivo em uma IIFE (incluindo esse comentário).
 
-/*
-Crie uma variável chamada `once`, que recebe `false` como seu valor.
-Crie um loop que execute enquanto essa variável for verdadeira. Dentro do
-loop, mostre no console a mensagem:
-'Entrou ao menos uma vez!'
-Qual loop você deve usar para que essa mensagem seja mostrada no console?
-*/
-// ?
+	/*
+	Crie uma variável chamada `once`, que recebe `false` como seu valor.
+	Crie um loop que execute enquanto essa variável for verdadeira. Dentro do
+	loop, mostre no console a mensagem:
+	'Entrou ao menos uma vez!'
+	Qual loop você deve usar para que essa mensagem seja mostrada no console?
+	*/
 
-/*
-Crie um objeto chamado `person`, que receba as seguintes propriedades:
-- 'name', 'age', 'weight' e 'birthday'. Preencha com os valores corretos
-para o nome, idade, peso e data de nascimento dessa pessoa.
-*/
-// ?
+	var once = false;
+	do{
+		console.log('Entrou ao menos uma vez');
+		break;
```

Comment:
-1
Não precisa do `break` aqui. Após entrar uma vez, e o `while` for avaliado, vendo que a variável `once = false`, ele segue o código em frente e nem entra mais aqui ;)